src/runtime-helpers.js

```diff
@@ -757,10 +807,10 @@ async function deployPackage (entities, ow, logger) {
     logger(`Info: action [${action.name}] has been successfully deployed.\n`)
   }
 
-  for (const api of entities.apis) {
-    logger(`Info: Deploying api [${api.name}]...`)
-    await ow.routes.create(api)
-    logger(`Info: api [${api.name}] has been successfully deployed.\n`)
+  for (const route of entities.apis) {
+    logger(`Info: Deploying route [${route.name}]...`)
```

Comment:
In `manifest.yml` we defines apis, so we should still log message as "Info: Deploying api ..." instead of calling it route. WDYT?

Reply:
would you be ok with `API route` ?